### PR TITLE
Education forum

### PR DESCRIPTION
**Pull request by erikrramsey**
Merging the changes from main into education forum, and publishing the first iteration of the education forum. Still ugly.

Navigation page - Links to quizzes. Adding browser page support soon.
Quiz page - quiz page linked to database backend.
Browser page - embedded content in the app.

Changes in server side to accommodate forum.

### CODE REVIEW

**Review by akummi — APPROVED** [review]
After reviewing the changes made in your PR, I am pleased to inform you that it has been approved. Your work merging the changes from main into the education forum and publishing the first iteration of the education forum is commendable, and it demonstrates your commitment to improving the application.

Your navigation page provides easy access to quizzes, and your plans to add browser page support soon will further enhance the user experience. The quiz page is linked to the database backend, which improves the application's stability and reliability. Additionally, your changes to the server-side to accommodate the forum are well thought out and executed.

**Comment by SnellJ2** [comment]
Approved

File: Carbon/navigation/screens/Forum/ForumScreen.js [hunk 1]
```diff
@@ -1,26 +1,122 @@
...
+                >
+                    <Text>
+                        {item.type}
+                    </Text>
+                </TouchableOpacity>
+            </View>
+        )
+    }
+
+    useEffect(() => {
```
Comment (SnellJ2) [inline Carbon/navigation/screens/Forum/ForumScreen.js:55]: Please comment above the effect what it does

**Review by SnellJ2 — APPROVED** [review]
Thank you for sharing this PR. In order to effectively communicate the significance of these changes, it would be helpful to provide more information on what you changed in the write up.
Going through the code I got a better understanding. -Jonas

**Pull request merged**